src/uu/cp/src/cp.rs

```diff
@@ -780,7 +780,13 @@ impl CopyMode {
         {
             Self::Update
         } else if matches.get_flag(options::ATTRIBUTES_ONLY) {
-            Self::AttrOnly
+            if matches.get_flag(options::REMOVE_DESTINATION) {
+                // if we pass --remove-destination --attributes-only, we want to move the copy mode
+                Self::Copy
+            } else {
+                // if no --remove-destination, back to the previous mode
+                Self::AttrOnly
+            }
```

Comment:
With the first comment I don't understand what it is trying to say with "we want to move the copy mode". And the second comment mentions a "previous mode" that doesn't exist. So my suggestion is to remove the comments.
```suggestion
            if matches.get_flag(options::REMOVE_DESTINATION) {
                Self::Copy
            } else {
                Self::AttrOnly
            }
```